x-pack/plugin/logstash/src/main/java/org/elasticsearch/xpack/logstash/Logstash.java

```diff
@@ -28,7 +28,9 @@
  */
 public class Logstash extends Plugin implements ActionPlugin {
 
-    private static final String LOGSTASH_TEMPLATE_NAME = "logstash-index-template";
+    private static final String LOGSTASH_TEMPLATE_FILE_NAME = "logstash-management";
+    private static final String LOGSTASH_INDEX_NAME = ".logstash-management";
```

Comment:
This is a good change, but may cause a problem on the Logstash side. Can you reach out and check?

Reply:
Yes, I'll reach out to double check, although this variable is misnamed currently - I thought I changed this, it should be `LOGSTASH_INDEX_TEMPLATE_NAME`.  This only changes the name of the template, not the name of the index/`index_pattern` in the template, which is still `.logstash`, so I don't *think* it would have much impact. I'll fix the name as well.

Reply:
@ycombinator would probably be the best person ask. This may also impact Kibana since this is the index template for the [Logstash centralized management](https://www.elastic.co/guide/en/logstash/6.5/logstash-centralized-pipeline-management.html).

Reply:
I can confirm that changing the name of the template (but not the index pattern it matches) will not break the Logstash Management UI in Kibana. As long as the logstash management index name remains `.logstash`, everything in the Kibana Logstash Management UI should continue to work.

Reply:
@robbavey Confirmed via Slack that renaming the index template should not impact Logstash management as long as the index itself stays named `.logstash`.  I'll also test this manually to make sure.